package.json

```diff
@@ -26,7 +26,7 @@
     "setup": "npm run npm:install:global && npm run dotnet:dev-certs:https",
     "setup:ci": "npm run npm:install:global",
     "postinstall": "npm run dotnet:restore",
-    "start": "npm run dotnet:run:Release:Tailspin.SpaceGame.Web",
+    "start": "npm run dotnet:run:Debug:Tailspin.SpaceGame.Web",
```

Comment:
`npm start` is for local development, so it makes more sense to use `Debug` instead of `Release`